### ISSUE TITLE
Cannot ignore custom getters while mapping

### ISSUE THREAD

**Issue by thombergs** [remondis-it/remap#63]
I have a class with a list of objects:

```java
class Foo {
  private List<Bar> bars;
  // getters and setters
}
```

Now I add a custom getter to return a filtered part of the `bars` list:
```java
class Foo {
  private List<Bar> bars;

  public List<Bar> getFilteredBars(){
    return this.bars.stream()
      .filter(...)
      .collect(Collectors.toList());
  }

  // more getters and setters
}
```

I want this getter to be ignored in mappings.

However, if I use `omitInSource(Foo:getBars)`, I get the following exception:

```
Caused by: com.remondis.remap.MappingException: The get-method for property '...' in type ... is not a valid Java Bean property.
```

This makes it impossible to add custom "helper" getters, which is kind of a hard restriction.

**Comment by schuettec** [comment]
Hi Tom, thanks for reporting.

I cannot reproduce this bug. Please have a look at #64 if it matches your scenario.

You can double check the ReMap version you are using. In 4.0.1 everything seems to work. Maybe we fixed the issue with #46.

**Comment by schuettec** [comment]
Not reproducable. Please update to the latest available version. I added a regression test just to be sure.
